powersimdata/scenario/analyze.py

```diff
@@ -209,7 +209,7 @@ def get_load_shed(self):
             # It's either on the server or in our local ScenarioData folder
             output_data = OutputData(data_loc=self.data_loc)
             load_shed = output_data.get_data(scenario_id, "LOAD_SHED")
-        except FileNotFoundError:
+        except OSError:
```

Comment:
Is this the error raised by the `check_file_exists` method of the `DataAccess` class?

Reply:
Exactly.